packages/docs/src/components/DocsLink.vue

```diff
@@ -5,39 +5,66 @@
       inline
       tag="span"
       v-if="preText"
-      :value="$te(preText) ? $t(preText) : preText"
+      :value="$t(preText)"
```

Comment:
Sorry, what was the reasoning here?

Reply:
```$t(path)``` returns 'path' translate (if translation exist in locale) or 'path' itself (if no translation in locale).
Therefore this code ```$te(preText) ? $t(preText) : preText``` is redundant

Reply:
That's not true, because you also get a warn.